src/schemaType/__tests__/EmbeddedType.test.ts

```diff
@@ -80,24 +80,26 @@ describe('validate', () => {
 	const valueSchema = new Schema(definition);
 	const embeddedType = new EmbeddedType(valueSchema);
 
-	test('should return an array of the errors encountered when validating the subdocument', async () => {
+	test('should return a map of the errors encountered when validating the subdocument', async () => {
 		const originalRecord: MvRecord = ['unrelated'];
 		const value = Document.createSubdocumentFromRecord(valueSchema, originalRecord);
 
 		const validationResults = await embeddedType.validate(value);
-		expect(validationResults).toHaveLength(2);
-		validationResults.forEach((result) => {
-			expect(result).toBe('Property is required');
-		});
+		expect(validationResults).toEqual(
```

Comment:
You're using the `const expected...` pattern elsewhere. It would be good to be consistent here.